src/diet.ts

```diff
@@ -342,130 +240,308 @@ function pillCheck(): void {
   }
 }
 
+const saladFork = $item`Ol' Scratch's salad fork`;
+const frostyMug = $item`Frosty's frosty mug`;
+const spleenCleaners = $items`extra-greasy slider, jar of fermented pickle juice`;
 const stomachLiverCleaners = new Map([
   [$item`spice melange`, [-3, -3]],
   [$item`synthetic dog hair pill`, [0, -1]],
   [$item`cuppa Sobrie tea`, [0, -1]],
 ]);
-export function computeDiet(): [MenuItem[], number][] {
-  // Roughly estimate how much spleen we need to clear for synth.
-  const expectedTurns = estimatedTurnsWithOrgans([]);
-  const spleenNeeded = Math.ceil((expectedTurns - haveEffect($effect`Synthesis: Greed`)) / 30);
-  const additionalSpleenNeeded = Math.max(
-    0,
-    spleenNeeded - Math.max(0, spleenLimit() - mySpleenUse()) - (3 - get("currentMojoFilters"))
-  );
-  const spleenCleanersNeeded = Math.ceil(additionalSpleenNeeded / 5);
-  const spleenCleaner = argmax(spleenCleaners.map((item) => [item, mallPrice(item)]));
 
-  const embezzlers = embezzlerCount();
-  const helpers = [Mayo.flex, $item`Special Seasoning`, saladFork, frostyMug];
-  const menu = [
+/**
+ * Generate a basic menu of high-yield items to consider
+ * @returns basic menu
+ */
+function menu(): MenuItem<string>[] {
+  const spaghettiBreakfast =
+    have($item`spaghetti breakfast`) &&
+    myFullness() === 0 &&
+    get("_timeSpinnerFoodAvailable") === "" &&
+    !get("_spaghettiBreakfastEaten")
+      ? 1
+      : 0;
+
+  return [
     // FOOD
-    new MenuItem($item`Dreadsylvanian spooky pocket`),
-    new MenuItem($item`tin cup of mulligan stew`),
-    new MenuItem($item`glass of raw eggs`, { maximum: availableAmount($item`glass of raw eggs`) }),
-    new MenuItem($item`Tea, Earl Grey, Hot`),
-    new MenuItem($item`meteoreo`),
-    new MenuItem($item`ice rice`),
-    new MenuItem($item`frozen banquet`),
-    new MenuItem($item`fishy fish lasagna`),
-    new MenuItem($item`gnat lasagna`),
-    new MenuItem($item`long pork lasagna`),
+    new MenuItem<string>($item`Dreadsylvanian spooky pocket`),
```

Comment:
you don't need the explicit type parameters here - TS can infer from the return value.